subgraph/bin/abis.js

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+
+const path = require('path')
+const fs = require('fs-extra')
+const abis = require('@unlock-protocol/contracts')
+const ethers = require('ethers')
+
+const abisFolderPath = path.join(__dirname, '..', 'abis')
+
+const unlockVersions = ['v11']
```

Comment:
Hum... is this going to work for older locks created on older versions of Unlock though?

Reply:
Hmmm good point